x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -417,11 +440,25 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
             features: [FEATURE.casesRead],
           },
           [SecurityPageName.caseConfigure]: {
-            features: [FEATURE.casesCrud],
+            features: [
+              FEATURE.casesCreate,
+              FEATURE.casesRead,
+              FEATURE.casesUpdate,
+              FEATURE.casesDelete,
+              FEATURE.casesPush,
+            ],
```

Comment:
This has been changed from `crud` to any of the cases permissions. We have only one direct array here, so the `OR` conditional is applied. Shouldn't we add another array so `AND` is applied?

Reply:
Oops! Great catch, thank you!